database-client/src/ceramicClient.ts

```diff
@@ -207,6 +207,28 @@ export class CeramicDatabase implements DataStorageBase {
     }
   }
 
+  // Update stamps by providerIds
+  async updateStamps(providerIds: PROVIDER_ID[]): Promise<void> {
+    this.logger.info(`updating stamp(s) on ${this.did}`);
+
+    const passport = await this.store.get("Passport");
+
+    // filter Passport stamp list by the platform's providerIds
+    if (passport && passport.stamps) {
+      const updatedStamps = passport.stamps.filter(stamp => !providerIds.includes(stamp.provider as PROVIDER_ID));
```

Comment:
As part of the filter, we could also delete the stamps streamId (`await deleteStamp(stamp.streamId)`) so that we're not leaving orphaned streams around?